Sources/Tracking/Service/HttpRequestRunner.swift

```diff
@@ -57,6 +58,33 @@ internal class UrlRequestHttpRequestRunner: HttpRequestRunner {
         }.resume()
     }
 
+    public func downloadFile(url: URL, onComplete: @escaping (URL?) -> Void) {
```

Comment:
is there a way for us to set a lifespan for cache items explicitly? or is that all managed by iOS under the hood

Reply:
I changed from using the system cache directory to system tmp directory. Better place to store these files since `cache` must be managed by us I believe.

Reply:
I also added a `cleanup` function when a push notification is touched or dismissed to delete images that we saved.